src/app.test.js

```diff
@@ -48,26 +48,10 @@ jest.mock('./utils', () => {
   }
 })
 
-jest.mock('jwt-decode', () => {
-  const originalModule = jest.requireActual('jwt-decode')
-  return {
-    ...originalModule,
-    jwtDecode: jest.fn()
-  }
-})
-
-jest.mock('node-cache', () => {
-  return jest.fn(() => {
-    return {
-      get: jest.fn(() => {}),
-      set: jest.fn(() => {})
-    }
-  })
-})
-
 // Tests
 beforeEach(() => {
   jest.resetAllMocks()
+  samUserCache.flushAll()
```

Comment:
I decided to use the actual cache in the unit tests and just flush it between each test